pages/index.tsx

```diff
@@ -0,0 +1,560 @@
+        One of the core features for effective data design in Avo is branched
+        workflow. It allows you to see the changes for a single feature and
+        provides implementation instructions that you can share with the
+        developers working on the analytics implementation
+      </p>
+      <div className={styles.row}>
+        <div style={{ display: 'flex' }}>
+          <b>Data managers</b>
+        </div>
+      </div>
+      <div className={styles.row}>
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="How to create a branch, view branch diff and share implementations instructions"
+          description=""
+          callToAction={new CallToAction('Learn', '/workspace/branches')}
+        />
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="Invite your coworkers to Avo"
+          description=""
+          callToAction={new CallToAction('Learn', '/workspace/members')}
+        />
+      </div>
+      <div className={styles.row}>
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="Discussions and collaboration on branches"
+          description=""
+          callToAction={
+            new CallToAction('Learn', '/data-design/start-collaborating')
+          }
+        />
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="Review and approval workflows (steps 4 and 5 in the guide)"
+          description=""
+          callToAction={
+            new CallToAction(
+              'Learn',
+              '/data-design/day-to-day-workflow#a-namestep-4-ask-for-reviewa-step-4--ask-for-review',
+            )
+          }
+        />
+      </div>
+      <H3>Publish tracking plan from Avo to other analytics tools</H3>
+      <p className={styles.text}>
+        You can set up your tracking plan to be automatically published to
+        Segment Protocols, Amplitude and Mixpanel every time a branch is merged.
+        We also provide a webhook for advanced automation.
+      </p>
+      <div className={styles.row}>
+        <div style={{ display: 'flex' }}>
+          <b>Data managers</b>
+        </div>
+      </div>
+      <div className={styles.row}>
+        <PageLink
+          image={AnalyticsManagerIcon}
+          title="Setting up publishing and auto publishing to Segment, Amplitude and Mixpanel"
+          description=""
+          callToAction={
+            new CallToAction(
+              'Get started',
+              '/workspace/tracking-plan/publishing',
+            )
+          }
+        />
+      </div>
+      <h2 id="stage2">2. Better tracking validation</h2>
+      <p className={styles.text}>
+        Add a lightweight SDK to your code to learn how actual implementation
+        compares with the defined tracking plan. Get instant feedback on what
+        tracking is implemented when working on new features with the
+        Implementation status in your Avo workspace. Observe your tracking calls
```

Comment:
Implementation status in your Avo workspace does not work unless you have Avo function, right? At least for the next few weeks.